UI/FileSavePickeriOS/FileSavePickeriOS/FileSavePickeriOS.Shared/MainPage.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<Page
+    x:Class="FileSavePickeriOS.MainPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:FileSavePickeriOS"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+        <Button Click="SaveFile_Click" HorizontalAlignment="Center" VerticalAlignment="Center">Save as...</Button>
```

Comment:
Resource-it so it could be Localized?

Reply:
This button is not part of the extension yet, it only triggers the `FileSavePicker`, so I suppose it does not need to be localized specifically

Reply:
it's a sample not a production app - localization is overkill here